modules/core/shared/src/main/scala/lucuma/core/util/Gid.scala

```diff
@@ -47,37 +47,43 @@ final class Gid[A](
   val tag:        Char Refined Letter,
   val isoPosLong: Iso[A, PosLong]
 ) extends BoundedEnumerable[A]
-     with Order[A]
-     with Show[A]
-     with Encoder[A]
-     with Decoder[A] {
+    with Order[A]
+    with Show[A]
+    with Encoder[A]
+    with Decoder[A] {
 
   // We use this in a few places
   private final val TagString: String = tag.value.toString
 
+  private final val AnchoredRegex: Regex = raw"$TagString-([1-9a-f][0-9a-f]*)".r
+
+  final val regex: Regex = AnchoredRegex.unanchored
```

Comment:
Huh, I didn't realize it was anchored by default.  Why do we need the unanchored version?

Reply:
We are going to pass the regex to `scalajs-react`'s router, which will use it to build a larger regex to parse the URL.

However, I just checked and said router actually needs the regex pattern as a `String`, so I'm going to slightly modify the PR.